README.md

```diff
@@ -11,7 +11,8 @@ For the MIPS official toolchain see
 	$ make CROSS_COMPILE=<toolchain-prefix> pistachio_marduk_defconfig
 	$ make CROSS_COMPILE=<toolchain-prefix>
 
-This will generate u-boot-pistachio-nor.img
+This will generate u-boot-pistachio_<board_name>-<version>.img
+eg. u-boot-pistachio_marduk-2015.10-v1.0.4.img
```

Comment:
Also mention that both conditions. i.e. if there are extra commits above the tag then it will add githash. 
e.g. u-boot-pistachio_marduk-2015.10-g75cbe38.img

May be version command output would be useful to add.